install-manuals/all-services/README.md

```diff
@@ -1,20 +1,24 @@
 # How to run all services
 
-All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
-
-We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
-
-
 ## Prerequisite
 
 ### Install docker
 
-[Istall Docker](https://docs.docker.com/engine/install/)
+[Install Docker](https://docs.docker.com/engine/install/)
 
-#### Docker Desktop
+#### Docker Desktop (not recommended for production use)
 
 [Install Docker Desktop on windows](https://docs.docker.com/desktop/windows/install/)
 
 [Install Docker Desktop on linux](https://docs.docker.com/desktop/linux/install/)
 
 [Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
+
+## Run
+
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file. Make sure you match the 
```

Comment:
you match the?

Reply:
Oh seem like I started a sentence that zou need to match the requirements but decided then to change the order